src/linagora.esn.unifiedinbox/app/components/sidebar/attachment-button/sidebar-attachment-button.pug

```diff
@@ -1,6 +1,6 @@
 button.btn.btn-link.btn-icon.visible-xl.waves-notransition.active(ng-if="$root.inbox.rightSidebar.isVisible", ui-sref="^", title="{{ 'Hide emails with attachments' | translate }}")
   i.mdi.mdi-paperclip
-button.btn.btn-link.btn-icon.visible-xl.waves-notransition(ng-if="!$root.inbox.rightSidebar.isVisible", ui-sref=".attachments", title="{{ 'Show emails with attachments' | translate }}")
+button.btn.btn-link.btn-icon.visible-xl.waves-notransition(ng-if="!$root.inbox.rightSidebar.isVisible", title="{{ 'Show emails with attachments' | translate }}", contextual-sidebar, data-content-template="/unifiedinbox/views/components/sidebar/attachment/sidebar-attachment.html", data-placement="right", data-controller="inboxListSidebarAttachmentController", data-controller-as="ctrl")
```

Comment:
Now we need to display the sidebar in the right if it's a medium and xl device and like the current behavior when it's a small device. You can check how it's done in `Calendar planing view` 


![Peek 04-11-2020 08-58](https://user-images.githubusercontent.com/11298249/98084599-583b6e80-1e7c-11eb-9753-3753d5d2ba42.gif)